src/load_tests/main.cc

```diff
@@ -7,9 +7,42 @@
 #include "check.h"
 #include "provision.h"
 #include "sslinit.h"
+#ifdef BUILD_OSTREE
+#include "treehub.h"
+#endif
 
 namespace bpo = boost::program_options;
 
+#ifdef BUILD_OSTREE
+void fetchRemoteCmd(const std::vector<std::string> &opts) {
+  std::string inputDir;
+  std::string outDir;
+  std::string branchName;
+  std::string remoteUrl;
+  uint opsPerSec;
+  uint opsNr;
+  uint parallelism;
```

Comment:
Can we change this to a standard type? On my machine, I found `typedef unsigned int uint;` in `/usr/include/sys/types.h`, where it is marked `Old compatibility names for C types.` and is surrounded by a `#ifdef __USE_MISC` guard. I'd rather not take our chances on the portability of this type.